proxy/src/managers/in_mem/keys.rs

```diff
@@ -0,0 +1,14 @@
+use std::{collections::HashMap, sync::Arc};
+
+use sam_common::{api::SignedEcPreKey, AccountId};
+use tokio::sync::Mutex;
+
+use crate::managers::{entities::Seed, DenimKeyManager};
+
+#[derive(Debug, Default, Clone)]
+pub struct InMemoryDenimKeyManager {
+    _seeds: Arc<Mutex<HashMap<AccountId, Seed>>>,
+    _signed_pre_keys: Arc<Mutex<HashMap<AccountId, SignedEcPreKey>>>,
+}
```

Comment:
can you use the inmem keymanager from sam server and the key generation from sam client?

Reply:
and if not, what should be made different on sam to actually do it